src/bootloader/bootloader.c

```diff
@@ -713,7 +713,7 @@ static size_t _api_versions(uint8_t* output)
         (const uint8_t*)&data->fields.signing_pubkeys_version,
         sizeof(version_t));
     _report_status(OP_STATUS_OK, output);
-    return BOOT_OP_LEN + sizeof(version_t) * 3;
+    return BOOT_OP_LEN + sizeof(version_t) * 2;
```

Comment:
naked number 2 should be a define.

Reply:
a define? not sure how it helps, the two versions are written out just before.

maybe the `len += ...` pattern after each write is more robust.